addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -146,13 +147,21 @@ var LinkDialog = Dialog.extend({
         if (this.data.iniClassName) {
             this.$('input[name="link_style_color"], select[name="link_style_size"] > option, select[name="link_style_shape"] > option').each(function () {
                 var $option = $(this);
-                if ($option.val() && self.data.iniClassName.match(new RegExp('(^|btn-| |btn-outline-)' + $option.val()))) {
-                    if ($option.is("input")) {
-                        $option.prop("checked", true);
-                    } else {
-                        $option.parent().find('option').removeAttr('selected').removeProp('selected');
-                        $option.parent().val($option.val());
-                        $option.attr('selected', 'selected').prop('selected', 'selected');
+                if ($option.val()) {
```

Comment:
Detail but you have the possibility to not indent the code you don't touch at no code readability cost and thus simplify the diff:

```
if (!value) {
    return;
}
```
(even without the diff simplification in mind, that would be my preferred form here to not stack "if" too much)